.github/workflows/frontend-hosting-pull-request.yml

```
@@ -65,11 +65,20 @@ jobs:
     steps:
       - uses: actions/checkout@v6
       - run: npm ci
-      - uses: FirebaseExtended/action-hosting-deploy@v0
+      - name: Compute Firebase preview channel
+        id: firebase-channel
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)